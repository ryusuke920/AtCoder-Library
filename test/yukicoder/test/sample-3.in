999 12345678987654321
