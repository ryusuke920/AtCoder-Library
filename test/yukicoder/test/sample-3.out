998244353
877641551
